Proposed_First_FLEDGE_OT_Details.md

```diff
@@ -67,6 +67,17 @@ The FOT#1 will include event-level reporting for both winning and losing bids. I
 
 If the bid being generated or scored loses the auction, the URL will be fetched. These worklets may also call a `forDebuggingOnly.reportAdAuctionWin()` API which operates similarly to `forDebuggingOnly.reportAdAuctionLoss()` API but only fetches the URL after a winning bid or score.
 
+If a seller rejected a bid from participating in an auction, the seller can optionally add a rejectReason field to its return object. The buyer can learn about the reject reason through calling `forDebuggingOnly.reportAdAuctionLoss()` API in  `generateBid()`, by using “${rejectReason}” in the report URL's query string. A component auction's bidders only get reject reasons from its component seller, but not reject reasons from top-level seller. The reject reason returned by `scoreAd()` should be one of
```

Comment:
```suggestion
If a seller rejects a bid from participating in an auction, the seller can optionally add a `rejectReason` field to its `scoreAd()` return object to convey to the bidder a more detailed reason why the bid was rejected. The buyer can learn about the reject reason by calling the `forDebuggingOnly.reportAdAuctionLoss()` API in  `generateBid()` and including `${rejectReason}` in the report URL's query string. A component auction's bidders only get reject reasons from its component seller, but not reject reasons from the top-level seller. The reject reason returned by `scoreAd()` should be one of
```

Reply:
I used https://omahaproxy.appspot.com/ to search my CL's commit, and it shows "initially landed in 108.0.5315.0".

Reply:
Done.
I searched my CL's commit in https://omahaproxy.appspot.com/, and it shows "initially landed in 108.0.5315.0".